src/push/monitor.ts

```diff
@@ -180,6 +180,7 @@ export async function createLightweightMonitors(
     const parsedDoc = parseDocument(content, {
       lineCounter,
       keepSourceTokens: true,
+      merge: true,
```

Comment:
Why is this change required?

Reply:
i will revert this. i was testing yaml parsing.